src/arns.lua

```diff
@@ -40,6 +39,18 @@ NameRegistry = NameRegistry or {
 --- @field target string|nil The address of the target of the reserved record
 --- @field endTimestamp number|nil The time at which the record is no longer reserved
 
+--- @alias ReturnedNameInitiator string -- The address of the initiator of the returned name
```

Comment:
Nit: you could use WalletAddress instead of string or just omit this alias altogether and use WalletAddress.